ui/packages/consul-hcp/app/components/consul/hcp/home/index.hbs

```diff
@@ -0,0 +1,8 @@
+<div
+  class={{'consul-hcp-home'}}
```

Comment:
do we need the curlies here?

Reply:
These are necessary so I removed them, gonna go ahead and merge